tests/Moq.Tests/StubExtensionsFixture.cs

```diff
@@ -185,10 +185,12 @@ public abstract class WriteOnlyProperty
 			public abstract string Test { set; }
 		}
 
-		[Fact]
-		public void Write_only_property_should_be_ignored_by_SetupAllProperties()
+		[Theory]
+		[InlineData(MockBehavior.Strict)]
+		[InlineData(MockBehavior.Loose)]
+		public void Write_only_property_should_be_ignored_by_SetupAllProperties(MockBehavior mockBehavior)
```

Comment:
This test's name no longer seems right. (If `SetupAllProperties` truly ignored write-only properties, then setting such a property *would* fail for strict mocks.)